### DISCUSSION TITLE
Does InlineHilite provide default format functions like SuperFences

### DISCUSSION THREAD

**Discussion by CrossDark** [facelessuser/pymdown-extensions#2531] [Q&A] [answered]
I've overridden the base inline logic to my own ones

```python
InlineHiliteExtension(
        custom_inline=[
            {
                'name': '*',
                'class': 'block',
                'format': variable_formatter,
            },
        ]


def variable_formatter(source, language, css_class, md):
    if language != '':
        """
        调用默认格式化函数
        Call the default format function.
        """
    match tuple(source):
        case '{', '#', *archers, '}':  # 匹配到{#锚点}
            archer = ''.join(archers)
            return f'<span id="{archer}">{archer}</span>'
        case _:
            return f'<code>{source}</code>'
```

if language has been defined, I want to call the default  format function, but I only find `#!py3  InlineHilitePattern.highlight_code()` in the source code which is an instance method. How can I call it in my own formatter or there are other formatters

**Comment by facelessuser** [comment, marked as answer]
You can try calling the following function in your custom formatter:

```py
el = md.inlinePatterns['backtick'].handle_code(lang, src)
```

This will find the appropriate language formatter that you wish to not handle.

You could use this method to call `highlight_code` instead if you know you want to call the highlighter directly.

**Reply by CrossDark** [reply]
Thanks, now my code can be rendered
